src/software/parameter/config/full_system_main_command_line_args.yaml

```diff
@@ -27,3 +27,9 @@ FullSystemMainCommandLineArgs:
         description: >-
             The directory to replay logged data from, if the 'replay' backend is selected. This must
             be the `SensorMsg` folder outputted by `proto_log_output_dir`.
+    logging_dir:
+        type: "std::string"
+        default: ""
+        description: >-
+            The directory to output logs to. Logs will not be outputted if this argument
```

Comment:
I don't think relative paths work so it would be good to document that this needs to be an absolute path

Reply:
added